common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -257,131 +276,167 @@ div.video {
         }
       }
 
-      ul.vcr {
-        float: left;
-        list-style: none;
-        margin: 0 lh() 0 0;
-        padding: 0;
+      .video-control,
+      .speeds,
+      .volume,
+      .size-quality-controls,
+      .language-control,
+      .branding,
+      .label,
+      .value,
+      .size-control,
+      .add-fullscreen {
+        height: 46px;
+        line-height: 46px;
+      }
 
-        @media (max-width: 1120px) {
-          margin-right: lh(0.5);
-          font-size: em(14);
-        }
+      .video-playback-controls {
+        @include clearfix();
+        background: #181818;
+        color: #f2f2f2;
 
-        li {
+        .video-controls-options {
+          display: block;
           float: left;
-          margin-bottom: 0;
+          margin: 0;
+          padding: 0;
+          list-style: none;
+          border-right: 1px solid #666;
+
+          .video-control {
+            display: block;
+            float: left;
+            margin: 0;
+
+            a {
+              @extend %video-button;
+              width: 14px;
+              padding: 0 lh(.75);
```

Comment:
Can we do away with this lh and maybe just give it an accurate width? Not a blocker, but would be nice...